tests/functional/centos/7/multiscenario-non-container/vagrant_variables.yml

```diff
@@ -0,0 +1,74 @@
+---
+
+# DEPLOY CONTAINERIZED DAEMONS
+docker: false
+
+# DEFINE THE NUMBER OF VMS TO RUN
+mon_vms: 1
+osd_vms: 6
```

Comment:
I think we could use only 2 osds nodes with a lot of disks (the second node would be used for osd_auto_discovery scenarios)

Reply:
I think if we wanna test filesture, bluestore, and osd_auto_discovery we need 3 nodes (because you can only do one osd_objectsore per host). I will make this change and we can discuss it